src/fs/rock/RockRebuild.h

```diff
@@ -81,7 +84,7 @@ class Rebuild: public AsyncJob, private IndependentRunner
     bool importEntry(Ipc::StoreMapAnchor &anchor, const sfileno slotId, const DbCellHeader &header);
     void freeBadEntry(const sfileno fileno, const char *eDescription);
 
-    void failure(const char *msg, int errNo = 0);
+    void failure(const char *msg, int errNo);
```

Comment:
Sorry, let's restore this declaration. This change is correct, but it is out-of-scope and may complicate understanding of this PR actual impact.

```suggestion
    void failure(const char *msg, int errNo = 0);
```

Reply:
Restored.